src/plan/global.rs

```diff
@@ -703,6 +703,7 @@ impl<VM: VMBinding> CommonPlan<VM> {
         trace: &mut T,
         object: ObjectReference,
     ) -> ObjectReference {
+        debug!("plan.global.706({:?})", object);
```

Comment:
This line should be either removed, or changed to a `trace` level logging (and the message needs to be changed as well to be meaningful).